### PR TITLE
Week 9 - Music Releases Project - Jamie and Tara

### PR DESCRIPTION
**Pull request by the-j-curl**
Link to my deployed project:
https://jt-music.netlify.app/

My reflections on how this week's project turned out:
1) I enjoyed learning about React and creating a site using components. 
2) I feel that the basics were okay to grasp, but then when you start using props, maps etc it becomes more confusing and harder to follow.
3)  I would of liked a better understanding of React before moving on to the next project.
4) I managed to get some help from my team mates when I got stuck on displaying the different artist names

Things I'd like to have clarified or explained in more detail:
1) I would like a better understanding of NPM and how it works, and also what the package.json file does.
2) More clarification on props, map and their purpose
3) The cleanest way to structure components and their CSS

### CODE REVIEW

File: code/src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export const App = () => {
+  return (
+    <>
+      <Header />
+      <section className="album-wrapper">
+        {albumArray.map(album => {
+          return (
+            <Album
+              key={album.name}
```
Comment (EmOlsz) [inline code/src/components/App.js:17]: Over here you pass proper `key` value, good job :)

File: code/src/components/Artist.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const Artist = (props) => {
+   
+          if (props.index === 0) {
```
Comment (EmOlsz) [inline code/src/components/Artist.js:5]: Well done!

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,164 @@
...
+.artist-name :hover {
+  text-decoration: underline;
+}
+
+.artist-box {
+  display: flex;
+}
+
+.album-name a {
+  color: #ffffff;
```
Comment (EmOlsz) [inline code/src/index.css:67]: `color: #fff`

File: code/src/index.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import { App } from './App'
+import { App } from './components/App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+const Write = () => {
```
Comment (EmOlsz) [inline code/src/index.js:6]: Don't know what was the purpose of this component, but we shouldn't do things like that. We should pass `<App />` component directly into `ReactDOM.render()` method

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Well done project Jamie & Tara! Your code in general fulfils good practices.
Some things needs to be changed though - take a look on comments I left for you and provide changes. Hope they will help you in future projects :)

File: code/src/components/Album.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { MusicIcons } from './MusicIcons'
```
[the remark this answers is not in the document]
Comment (the-j-curl) [reply code/src/components/Album.js:2]: Cool, thanks for explaining this Maks. 
Changes have been made as requested.

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**